documentation/Installing-Clear-Containers-on-Fedora.md

```diff
@@ -1,4 +1,12 @@
-# Install the Clear Containers runtime on Fedora 25
+# Install the Clear Containers runtime on Fedora
+
+## Introduction
+
+Clear Containers 2.1 is available for Fedora versions **24** and **25**.
+Clear containers supports latest version of Docker CE (currently 17.05), with
```

Comment:
s/Clear containers supports latest version/Clear Containers supports the latest version/.